### PR TITLE
Fix alternative spelling

### PR DESCRIPTION
**Pull request by axsuul**
Very simple pull request here, the misspelling of alternative was bothering me.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/34305)

### CODE REVIEW

**Comment by axsuul** [comment]
I signed it!

**Comment by lavalamp** [comment]
Please squash the commits, thanks--otherwise lgtm

**Comment by axsuul** [comment]
@lavalamp Commits squashed, thanks

**Comment by lavalamp** [comment]
ok to test

lgtm, thanks!

**Comment by axsuul** [comment]
@k8s-bot gke e2e test this

File: vendor/github.com/magiconair/properties/doc.go [hunk 1]
```diff
@@ -102,7 +102,7 @@
 //   v = p.GetString("key", "def")
 //   v = p.GetDuration("key", 999)
 //
-// As an alterantive properties may be applied with the standard
+// As an alternative properties may be applied with the standard
```
Comment (wojtek-t) [inline vendor/github.com/magiconair/properties/doc.go]: Please do not modify vendor/ directory - those files are managed by godeps and we are not modifying them manually.

Comment (lavalamp) [inline vendor/github.com/magiconair/properties/doc.go]: +1, I can't approve it with this change

**Comment by bgrant0607** [comment]
Thanks but this appears to have been resolved by another PR.

**Pull request closed without merging**